boards/posix/native_posix/doc/board.rst

```diff
@@ -411,6 +411,16 @@ The following peripherals are currently provided with this board:
   Note that this device can only be used with Linux hosts, and that the user
   needs elevated permissions.
 
+**Bluetooth controller**:
+  It's possible to use the host's Bluetooth adapter as a Bluetooth
```

Comment:
Some formatting and wording tweaks:

    It's possible to use the host's Bluetooth adapter as a Bluetooth
    controller for Zephyr. To do this, the HCI device needs to be
    passed as a command line option to ``zephyr.exe``. For example,
    to use ``hci0``, use ``sudo zephyr.ext --bt-dev=hci0``.  Using
    the device requires root privileges (or the CAP_NET_ADMIN POSIX
    capability, to be exact)  so ``zephyr.exe`` needs to be through
    ``sudo``. The chosen HCI device must be powered down and
    support Bluetooth Low Energy (Bluetooth specification version
    4.0 or greater).

Reply:
fixed